lambda/create/main.go

```diff
@@ -104,6 +105,20 @@ func (l *Lambda) HandleEvent(ctx context.Context, req events.APIGatewayProxyRequ
 	data.Status = shared.LpaStatusProcessing
 	data.UpdatedAt = time.Now()
 
+	if data.Channel == shared.ChannelPaper && len(input.RestrictionsAndConditionsImages) > 0 {
+		data.RestrictionsAndConditionsImages = make([]shared.File, len(input.RestrictionsAndConditionsImages))
+		for i, image := range input.RestrictionsAndConditionsImages {
+			path := fmt.Sprintf("%s/scans/rc_%d_%s", data.Uid, i, image.Filename)
```

Comment:
Are we ever likely to get identical filenames?

Reply:
I think the inclusion of `i` should prevent that, since `{uid}/scans/rc_{i}` should itself be a unique identifier.

Reply:
Ah, missed that! All good then 👍